docs/modules/ROOT/pages/_partials/task-creating-fuse-integration.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// <List assemblies here, each on a new line>
+
+
+[id='creating-fuse-integration_{context}']
+= Creating an integration
+
+ifdef::location[]
+// tag::intro[]
+To use the Fuse connections, you create an integration in Red Hat Fuse Online.
+// end::intro[]
+endif::location[]
+
+
+. Select *Integrations* from the left hand menu.
+
+. Select the *Create Integration* button to start the *Create Integration* wizard.
+
+. Choose `Walkthrough One Messaging App` as the *Start Connection*.
+
+. When prompted to *Choose an Action*, select *Subscribe for messages*.
+
+. When prompted for a *Destination Name*, enter `work-queue/requests`.
```

Comment:
Nothing to do with this PR, but is there anything in Asciidoc that renders an easily copyable field? So they'd just have to click a button or something to save to their clipboard instead of copy the `work-queue/requests` part themselves?